__mocks__/mock_node.js

```diff
@@ -51,18 +46,22 @@ switch (process.env.CONFIG_NAME) {
     // If running in dev mode, use all available mocks
     //
     console.warn('--- Injecting All TC mocks ---');
-    mockAll();
-    enhancedModuleResolve();
+    useAllMocks(true);
     break;
   case 'devlive':
     //
     // Dev live is internally connected, but all external connections are mocked
     //
     console.warn('--- Injecting external TC mocks ---');
-    enhancedModuleResolve("googleapis", "google-auth-library");
+    useSomeMocks("googleapis", "google-auth-library");
     break;
   default:
-    enhancedModuleResolve();
+    //
```

Comment:
default applies to both prodtest/prodbeta/prod.

The libraries themselves select the appropriate implementation to include for the environment